ui/app/src/components/workspaces/WorkspaceOperationsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { DefaultPalette, IStackItemStyles, IStackStyles, Stack } from "@fluentui/react";
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Operation } from '../../models/operation';
+import { Workspace } from '../../models/workspace';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { HttpMethod, ResultType, useAuthApiCall } from '../../useAuthApiCall';
+import config from '../../config.json';
+
+
+
+interface WorkspaceOperationPanelProps {
+    workspace: Workspace
+}
+
+interface WorkspaceOperationPanelItemProps {
+    header: String,
+    val: String
+}
+
+const WorkspaceOperationPanelItem: React.FunctionComponent<WorkspaceOperationPanelItemProps> = (props: WorkspaceOperationPanelItemProps) => {
```

Comment:
Bit of a judgement call, but personally I think we could move this to a separate file - `ResourceOperationListItem` (let's not use 'panel' as Fluent's use of 'panel' will confuse us in future). This would help keep each component as small as possible... but again you could fall down either side on this :)

Reply:
Agree! Seperated `ResourceOperationsListItem` from the `ResourceOperationsList`. Panel is not more.
Thanks